lib/Controller/ApiV2ResponseTrait.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * Nextcloud - News
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author    Paul Tirk <paultirk@paultirk.com>
+ * @copyright 2020 Paul Tirk
+ */
+
+namespace OCA\News\Controller;
+
+use \OCP\AppFramework\Http;
+use \OCP\AppFramework\Http\JSONResponse;
+
+use \OCA\News\Db\IAPI;
+
+trait ApiV2ResponseTrait
```

Comment:
I'd rather see this as a function or argument in the existing trait. That way we can keep the APIs in sync a little better.

Reply:
I thought since the "old" API is probably going away in a few releases it would be better to separate this already into a new file and rename it afterwards, but of course I can move it into the other ones.